Description: Build against libmate-slab-dev as shipped in Debian (and as will be in MATE 1.10)
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/configure.ac
+++ b/configure.ac
@@ -42,7 +42,7 @@
 		gobject-2.0 			\
 		gtk+-2.0 >= $GTK_REQUIRED 	\
 		gdk-2.0				\
-		libslab >= $SLAB_REQUIRED"
+		mate-slab >= $SLAB_REQUIRED"
 
 
 PKG_CHECK_MODULES(APPLICATION_BROWSER, [ $COMMON_MODULES mate-desktop-2.0 unique-1.0 ])
--- a/main-menu/src/network-status-agent.c
+++ b/main-menu/src/network-status-agent.c
@@ -32,7 +32,7 @@
 #include <iwlib.h>
 #include <glibtop/netlist.h>
 #include <glibtop/netload.h>
-#include <libslab/slab.h>
+#include <libmate-slab/slab.h>
 
 typedef struct
 {
--- a/main-menu/src/hard-drive-status-tile.h
+++ b/main-menu/src/hard-drive-status-tile.h
@@ -21,7 +21,7 @@
 #ifndef __HARD_DRIVE_STATUS_TILE_H__
 #define __HARD_DRIVE_STATUS_TILE_H__
 
-#include <libslab/slab.h>
+#include <libmate-slab/slab.h>
 
 G_BEGIN_DECLS
 
--- a/main-menu/src/main-menu-ui.c
+++ b/main-menu/src/main-menu-ui.c
@@ -37,7 +37,7 @@
 #include <gio/gio.h>
 #include <gio/gdesktopappinfo.h>
 #include <unistd.h>
-#include <libslab/slab.h>
+#include <libmate-slab/slab.h>
 
 #include "hard-drive-status-tile.h"
 
--- a/main-menu/src/main-menu.c
+++ b/main-menu/src/main-menu.c
@@ -26,7 +26,7 @@
 #include <glib/gi18n.h>
 #include <mate-panel-applet.h>
 #include <string.h>
-#include <libslab/slab.h>
+#include <libmate-slab/slab.h>
 
 #include "main-menu-ui.h"
 
--- a/main-menu/src/network-status-tile.h
+++ b/main-menu/src/network-status-tile.h
@@ -21,7 +21,7 @@
 #ifndef __NETWORK_STATUS_TILE_H__
 #define __NETWORK_STATUS_TILE_H__
 
-#include <libslab/slab.h>
+#include <libmate-slab/slab.h>
 
 G_BEGIN_DECLS
 
--- a/main-menu/src/tile-table.h
+++ b/main-menu/src/tile-table.h
@@ -22,7 +22,7 @@
 #define __TILE_TABLE_H__
 
 #include <gtk/gtk.h>
-#include <libslab/slab.h>
+#include <libmate-slab/slab.h>
 
 G_BEGIN_DECLS
 
--- a/application-browser/src/application-browser.c
+++ b/application-browser/src/application-browser.c
@@ -27,7 +27,7 @@
 #include <stdlib.h>
 #include <glib/gi18n.h>
 #include <gio/gio.h>
-#include <libslab/slab.h>
+#include <libmate-slab/slab.h>
 
 #define APPLICATION_BROWSER_SCHEMA  "org.mate.gnome-main-menu.application-browser"
 #define NEW_APPS_MAX_ITEMS          "new-apps-max-items"
